client/my-sites/plans/jetpack-plans/types.ts

```diff
@@ -120,20 +120,20 @@ export interface SelectorProduct extends SelectorProductCost {
 	productSlug: string;
 	annualOptionSlug?: string;
 	monthlyOptionSlug?: string;
-	iconSlug: string;
-	type: ItemType;
+	iconSlug?: string;
+	type?: ItemType;
 	costProductSlug?: string;
 	monthlyProductSlug?: string;
-	displayName: TranslateResult;
-	shortName: TranslateResult;
+	displayName?: TranslateResult;
+	shortName?: TranslateResult;
 	subheader?: TranslateResult;
-	tagline: TranslateResult;
-	description: TranslateResult | ReactNode;
+	tagline?: TranslateResult;
+	description?: TranslateResult | ReactNode;
 	shortDescription?: TranslateResult | ReactNode;
 	children?: ReactNode;
-	term: Duration;
+	term?: Duration;
 	buttonLabel?: TranslateResult;
-	features: SelectorProductFeatures;
+	features?: SelectorProductFeatures;
```

Comment:
I'm going to try to extend the interface and use the extended rather than modify this one making fields optional.